src/matrix/net/common.ts

```diff
@@ -15,7 +15,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-export function encodeQueryParams(queryParams) {
+import {BlobHandle} from "../../platform/web/dom/BlobHandle.js";
+
+export interface IEncodedBody {
```

Comment:
I've had a think when to use `interface` and `type` (as we're using them interchangeably now) and have written the guidelines down here: https://github.com/vector-im/hydrogen-web/blob/master/doc/TS-MIGRATION.md#use-type-rather-than-interface-for-named-parameters-and-pojo-return-values

> 
```suggestion
export type EncodedBody = {
```